core/domain/classifier_domain_test.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, softwar
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for classifier domain objects."""
+
+from core.tests import test_utils
+from core.domain import classifier_domain
+
+class ClassifierDomainTests(test_utils.GenericTestBase):
+
+    def setUp(self):
```

Comment:
This can be dropped; it doesn't add anything since there's nothing in the method besides the call to the superclass's setUp().